core/src/main/java/dk/gtz/graphedit/view/EditorTabPaneController.java

```diff
@@ -94,5 +108,20 @@ private void initTabpaneBufferContainer() {
 	    if(c.wasRemoved())
 		c.getValueRemoved().getViews().forEach(v -> tabpane.getTabs().remove(v));
 	});
+
+	Platform.runLater(() -> {
```

Comment:
Extract this into a separate function for clarity. Out of context it can get a bit confusing